batchflow/dataset.py

```diff
@@ -273,8 +279,8 @@ def cv_split(self, method='kfold', n_splits=5, shuffle=False):
 
             setattr(self, 'cv'+str(i), deepcopy.copy(self))
             cv_dataset = getattr(self, 'cv'+str(i))
-            cv_dataset.train = self.create_subset(train_indices)
-            cv_dataset.test = self.create_subset(test_indices)
+            cv_dataset.train = self.create_subset(DatasetIndex(train_indices))
```

Comment:
It doesn't have to be a `DatasetIndex`. Why do you need to wrap it here?

Reply:
Because `create_subset` works only with `DatasetIndex`